app/services/asset_manager/asset_updater.rb

```diff
@@ -1,36 +1,30 @@
 class AssetManager::AssetUpdater
   include AssetManager::ServiceHelper
 
-  class AssetAlreadyDeleted < StandardError
-    def initialize(identifier, asset_data_id)
-      super("Asset '#{identifier}' for Attachment Data #{asset_data_id} expected not to be deleted in Asset Manager")
-    end
-  end
-
   class AssetAttributesEmpty < StandardError
-    def initialize(identifier, asset_data_id)
-      super("Attempting to update '#{identifier}' for Attachment Data #{asset_data_id} with empty attachment attributes")
+    def initialize(asset_manager_id)
+      super("Attempting to update Asset with asset_manager_id: '#{asset_manager_id}' with empty attachment attributes")
     end
   end
 
   def self.call(*args)
     new.call(*args)
   end
 
-  def call(asset_manager_id, asset_data, new_attributes = {})
-    update_with_asset_manager_id(asset_manager_id, asset_data, new_attributes)
+  def call(asset_manager_id, new_attributes)
+    update_with_asset_manager_id(asset_manager_id, new_attributes)
```

Comment:
Think there is no need for the other method here.

Reply:
How about removing the `update_with_asset_manager_id` itself?